lldb/tools/lldb-dap/.gitignore

```diff
@@ -0,0 +1,5 @@
+out
+node_modules
+.vscode-test
+*.vsix
+!.vscode
```

Comment:
Can we just create a `vscode` folder (no leading `.`) and remove this file? Why do we need `out` and `node_modlues` to be added to this? Does `npm` create temp files here for some reason?

Reply:
yes, npm creates all these files